### PR TITLE
Proto Emitter Gun now has its own name and description

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do

Gives the proto emitter its own name and description.

## Why It's Good For The Game
Knowing that it's a different item without figuring it out or code-diving is good.

## Testing
Looked at gun.

## Changelog
:cl:
tweak: Proto emitter has its own name and description.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -272,7 +272,9 @@
 /obj/item/gun/energy/emitter/cyborg/emp_act()
 	return
 
-/obj/item/gun/energy/emitter/cyborg/proto
+/obj/item/gun/energy/emitter/cyborg/proto	// Special version given by the Safety Override upgrade and as a standard module for ERT engiborgs.
+	name = "mobile proto-emitter"
+	desc = "An emitter removed from its base, and attached to a laser cannon frame. This one is running on unoptimised software, reducing its effectiveness."
```
Comment (Henri215) [inline code/modules/projectiles/guns/energy/laser.dm:277]:
```suggestion
	desc = "An emitter removed from its base and attached to a laser cannon frame. This one operates on unoptimised software, reducing its effectiveness."
```
think its better to read like that?

**Comment by Henri215** [comment]
Requires wiki update because the name there is different

File: code/modules/projectiles/guns/energy/laser.dm (hunk 1 above)
Comment (Fordoxia) [reply code/modules/projectiles/guns/energy/laser.dm:277]: Aye, probably.

File: code/modules/projectiles/guns/energy/laser.dm [hunk 2]
```diff
@@ -272,7 +272,9 @@
 /obj/item/gun/energy/emitter/cyborg/emp_act()
 	return
 
-/obj/item/gun/energy/emitter/cyborg/proto
+/obj/item/gun/energy/emitter/cyborg/proto	// Special version given by the Safety Override upgrade and as a standard module for ERT engiborgs.
```
Comment (lewcc) [inline code/modules/projectiles/guns/energy/laser.dm:275]:
```suggestion
/// Special version given by the Safety Override upgrade and as a standard module for ERT engiborgs.
/obj/item/gun/energy/emitter/cyborg/proto	
```

**Review by Henri215 — APPROVED** [review]
Lewcs good to Lewcs

**Review by S34NW — APPROVED** [review]

**Pull request merged**